tensorflow/training/docker/1.15.3/py3/Dockerfile.gpu

```diff
@@ -179,7 +179,7 @@ RUN ${PIP} install --no-cache-dir -U \
     mpi4py==3.0.2 \
     "cryptography>=2.3" \
     "sagemaker-tensorflow>=1.15,<1.16" \
-    sagemaker-tensorflow-training==10.1.0 \
+    "sagemaker-tensorflow-training>=10,<20" \
```

Comment:
Can we make this change for py36 image for TF1.15.3 as well ?

Reply:
Good call, just pushed.

Do any of the older TF versions need to consume the updated `sagemaker-tensorflow-training` package as well?

Reply:
Yes, TF2.0.2 and Tf2.1.1 will also need it, but we can build only one TF2 images at a time.